src/common/util/util.ts

```diff
@@ -423,3 +423,29 @@ export function memcpy(
 ): void {
   subarrayAsU8(dst.dst, dst).set(subarrayAsU8(src.src, src));
 }
+
+/**
+ * Used to create a value that is specified by multiplying some runtime value
+ * by a constant and then adding a constant to it.
+ */
+export interface SpecValue {
```

Comment:
This name seems too general for what it is, but I don't really have a better idea. `ValueTestVariant` maybe?

Reply:
done